packages/client/src/containers/SignIn/index.tsx

```diff
@@ -63,14 +64,20 @@ const SignIn = () => {
             maxWidth="100%"
             errors={errors}
             name="password"
-            label="Password"
             type="password"
             disabled={loading}
             autoComplete="current-password"
           />
-          <Button type="submit" fullWidth disabled={loading} margin="30px 0 0 0">
+          <Button type="submit" fullWidth disabled={loading} margin="30px 0" withArrow>
             Sign In
           </Button>
+          <Checkbox
+            required
+            ref={register({ required: true })}
+            errors={errors}
+            label="Keep me signed in."
+            name="keepSignIn"
```

Comment:
make sure that name match field in api endpoint

Reply:
@Mefjus for that moment I changed the field name but in a separate PR, we need to change the name of this variable in API. To be honest `keepMeLogin` is grammatically incorrect. Besides that, we must decide which form we will handle and by that, I mean `Sign In and Sign Up` or `Login and Registration`.